cmd/flux/create_image_policy.go

```diff
@@ -94,14 +98,27 @@ func createImagePolicyRun(cmd *cobra.Command, args []string) error {
 		policy.Spec.Policy.SemVer = &imagev1.SemVerPolicy{
 			Range: imagePolicyArgs.semver,
 		}
+	case imagePolicyArgs.alpha != "":
+		if imagePolicyArgs.alpha != "desc" && imagePolicyArgs.alpha != "asc" {
+			return fmt.Errorf("--select-alpha must be one of [\"asc\", \"desc\"]")
+		}
+		policy.Spec.Policy.Alphabetical = &imagev1.AlphabeticalPolicy{
+			Order: imagePolicyArgs.alpha,
+		}
 	default:
-		return fmt.Errorf("a policy must be provided with --semver")
+		return fmt.Errorf("a policy must be provided with either --select-semver or --select-alpha")
 	}
 
 	if imagePolicyArgs.filterRegex != "" {
 		policy.Spec.FilterTags = &imagev1.TagFilter{
 			Pattern: imagePolicyArgs.filterRegex,
 		}
+
+		if imagePolicyArgs.filterExtract != "" {
```

Comment:
Stretch goal: can the extract pattern be validated against the regex, e.g., to make sure it doesn't mention capture groups that aren't present?

Reply:
Added validation, though the amount of code to do this is a bit humorous

Reply:
Oof, yep. If extract is taken verbatim from `<go stdlib>/regexp/regexp.go`, it's worth noting this I think.